src/Common/Core/Impl/Versions/VsVersion.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Common.Core {
+    public static class VsVersion {
```

Comment:
On the other hand, this solution is VS solution so one can perceive this as version of the toolset.